clang/test/SemaSYCL/kernelname-enum.cpp

```diff
@@ -1,13 +1,14 @@
 // RUN: %clang_cc1 -fsycl -fsycl-is-device -fsycl-int-header=%t.h -fsyntax-only -Wno-sycl-2017-compat -verify %s
 
+// expected-error@Inputs/sycl.hpp:220 2{{kernel name is invalid. Unscoped enum requires fixed underlying type}}
 #include "Inputs/sycl.hpp"
 
 enum unscoped_enum_int : int {
   val_1,
   val_2
 };
 
-// expected-note@+1 {{'unscoped_enum_no_type_set' declared here}}
+// expected-note@+1 1+ {{'unscoped_enum_no_type_set' declared here}}
```

Comment:
Could you please use explicit number here?
```suggestion
// expected-note@+1 2{{'unscoped_enum_no_type_set' declared here}}
```